tests/metrics/storage/fio-k8s/README.md

```diff
@@ -0,0 +1,26 @@
+# FIO test in Kubernetes
+
+This is an automation to run `fio` with Kubernetes.
+
+## Requirements:
+- Kubernetes cluster running.
+- Kata configured as `runtimeclass`. 
+The test is structured as follow: 
```

Comment:
blank line needed before this line. If not, it ends being part of the previous line.

Also, I think this should be:
`## Test structure`
as it is not part of the Requirements section.